spacelift/data_github_enterprise_integration.go

```diff
@@ -0,0 +1,72 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+)
+
+var githubErpIntegrationFields = struct {
```

Comment:
Maybe use full name here because here because you are doing it for the other VCS providers as well.